modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/Util.kt

```diff
@@ -48,12 +47,7 @@ object Util {
         }
 
     private fun appInfoHasBoolean(key: String, context: Context, default: Boolean = false): Boolean {
-        val appInfo = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            context.packageManager.getApplicationInfo(context.packageName, ApplicationInfoFlags.of(PackageManager.GET_META_DATA.toLong()))
-        } else {
-            @Suppress("DEPRECATION")
-            context.packageManager.getApplicationInfo(context.packageName, PackageManager.GET_META_DATA)
-        }.metaData
+        val appInfo = context.packageManager.getApplicationInfo(context.packageName, PackageManager.GET_META_DATA).metaData
```

Comment:
The `Util.isAutomotive` method uses this, I logged it out with the Automotive emulator and it still seems to work correctly. 

> Error: Must be one or more of: PackageManager.GET_META_DATA, PackageManager.GET_SHARED_LIBRARY_FILES, PackageManager.MATCH_UNINSTALLED_PACKAGES, PackageManager.MATCH_SYSTEM_ONLY, PackageManager.MATCH_DISABLED_COMPONENTS, PackageManager.MATCH_DISABLED_UNTIL_USED_COMPONENTS, PackageManager.GET_DISABLED_UNTIL_USED_COMPONENTS, PackageManager.GET_UNINSTALLED_PACKAGES, PackageManager.MATCH_APEX [WrongConstant]
            context.packageManager.getApplicationInfo(context.packageName, ApplicationInfoFlags.of(PackageManager.GET_META_DATA.toLong()))